valley/classes.txt

```diff
@@ -100,143 +114,4 @@ new class script_variable = {
 			}
 		}
 	}
-	set = {
```

Comment:
adieu comparators,adieu operators

Reply:
that's cus I made a better system for them. Praise be & operators

Reply:
I'm blind ,where are the &operators?

Reply:
they were introduced a while ago, but I forgot to remove this system when I made them.
https://github.com/Kurash1/Arc-Valley/blob/bbe288a02002dc7b885b2718273e9e5d3e25a7c1/Compiler/Compiler.cs#L951